llvm/lib/Target/AMDGPU/AMDGPUInsertDelayAlu.cpp

```diff
@@ -331,6 +331,11 @@ class AMDGPUInsertDelayAlu {
     bool Changed = false;
     MachineInstr *LastDelayAlu = nullptr;
 
+    bool VALUSALUStall = false;
+    MCRegUnit lastSgprWrite = 0;
+    MCRegUnit longestWait = 0;
+    unsigned deletedCyclesNum = 0;
```

Comment:
In LLVM style variable names start with an upper case letter:
```suggestion
    MCRegUnit LastSgprWrite = 0;
    MCRegUnit LongestWait = 0;
    unsigned DeletedCyclesNum = 0;
```